openvdb/doc/changes.txt

```diff
@@ -10,9 +10,14 @@
 Bug fixes:
 - Fixed a bug in the @link tools/Clip.h clip@endlink tool that caused
   some grid metadata to be discarded.
+- Added a check to @vdblink::points::setGroup setGroup@endlink to compare
```

Comment:
What bug did this change address, specifically (data corruption, a crash, etc.)?

Reply:
perhaps an improvement is more relevant here as the unexpected behaviour only manifested as an incorrect runtime error:

    - Fixed point group initialization in points::setGroup() when a PointDataIndex
      tree is used but has been constructed from NaN positional values.